internals/search.py

```diff
@@ -125,20 +115,43 @@ def process_queriable_field(
 # A value that a feature field can be compared against.  It can be
 # a single word or a quoted string.
 VALUE_PATTERN = r'[^" ]+|"[^"]+"'
+# Logical operators.
+# TODO(kyleju): support 'OR' logic
+LOGICAL_OPERATORS_PATTERN = r'OR|-'
```

Comment:
I think that allowing `OR` here would capture the first two letters of a field name that begins with 'or'.   E.g., if we had a field called 'origin', then it would parse as ('or', 'igin').

Reply:
I will remove OR for now and think more carefully